ksqldb-functional-tests/src/test/resources/historical_plans/multi-joins_-_table-table-table_-_SR-enabled_key_format/6.2.0_1606950117788/plan.json

```diff
@@ -1,312 +0,0 @@
-{
```

Comment:
I'm not sure why Github is rendering this file as having been deleted. A new version is added back in the third commit in this PR (see https://github.com/confluentinc/ksql/pull/6749/commits/58da19f7b2128fbaa431021133af5f0594b3b043#diff-8f6edbed395aac8e9c455d95a9bee5855dcb0d8230b2b280d50093def7198380) but GitHub isn't displaying the new file for some reason?

Reply:
Same thing with the other "deleted" files in this PR. I'm hoping this is a Github bug and that the new files (added in the third commit) aren't being lost somehow.

Reply:
yeah, looks like the third commit adds things back. Also jenkins would fail if they weren't added back because of PlannedTestsUpToDatTest, right?

Reply:
That's the hope...! (Yes, I verified locally :) )